elasticsearch-transport/lib/elasticsearch/transport/transport/http/manticore.rb

```diff
@@ -63,7 +63,7 @@ def __build_connections
               @request_options[:headers] = options[:headers]
             end
 
-            client_options = setup_ssl(options[:ssl] || {})
+            client_options = {:ssl => options[:ssl] || {}}
```

Comment:
Hi, looking, have you pushed the updates, Tray? I'm good to merge when we add those minor things :)